### PR TITLE
Update SPMT-settings.md

### PR DESCRIPTION
**Pull request by dylanhayes**
Add examples of content type exclusion and the correct separator character between content types

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/SPMT-settings.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The following table details the settings available in SPMT.  Advanced settings s
...
 |Filter subsites|Enter any subsite names you want to exclude from migration.|
 |Filter lists and libraries|Enter the names of the lists and libraries you want to exclude from migration.|
-|Filter content type|Enter the content types you want to exclude from migration.|
+|Filter content type|Enter the content types you want to exclude from migration separated by ;. e.g. Document;My Document|
```
Comment (JohanFreelancer9) [inline migration/SPMT-settings.md:65]:
```suggestion
|Filter content type|Enter the content types you want to exclude from migration, separated by a semicolon (;), for example, Document;My Document.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dylanhayes, kindly commit the suggested change. Many thanks.

**Comment by JohanFreelancer9** [comment]
@dylanhayes, thank you for adding the suggested change, but I notice that you have closed the PR. Was that intentional? If not, could you please reopen it? Thanks.

**Comment by dylanhayes** [comment]
> @dylanhayes, thank you for adding the suggested change, but I notice that you have closed the PR. Was that intentional? If not, could you please reopen it? Thanks.

Sorry - accident due to doing too many thing at once at the end of the day.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**